sound/soc/sof/intel/hda-stream.c

```diff
@@ -466,9 +466,6 @@ irqreturn_t hda_dsp_stream_interrupt(int irq, void *context)
 	u32 rirb_status;
 	u32 status;
 
-	if (!pm_runtime_active(bus->dev))
-		return ret;
-
```

Comment:
yep for RIRB but for stream events its' not really possible to get stream interrupts without being in pm_runtime_active, is it?  Do we want to remove this completely?

Reply:
agree that we should just move it to after the RIRB status check. And also just fyi that this doesnt fix our IPC timeout issues still.